draft-ietf-quic-recovery.md

```diff
@@ -1588,23 +1592,26 @@ Invoked when an ACK frame with an ECN section is received from the peer.
 Invoked when DetectAndRemoveLostPackets deems packets lost.
 
 ~~~
-   InPersistentCongestion(lost_packets):
+   InPersistentCongestion(lost_packets, pn_space):
+     ack_delay = 0
+     if (pn_space == ApplicationData):
+       ack_delay = max_ack_delay
```

Comment:
Based on discussions about PC elsewhere, I realized my error.  This needs to be reverted.  In fact, I would suggest that we initialize max_ack_delay to 25ms for that reason.